src/bigartm/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ include_directories(${CMAKE_CURRENT_LIST_DIR}/../)
 
 set(SRC_LIST
```

Comment:
Before merging into master this change should be moved grom src/bigartm into src/artm/core (of course after refactoring). You should create test file for it in src/artm_test and test data into test_data directory.